### ISSUE TITLE
`calc_diversity` segfault

### ISSUE THREAD

**Issue by mmore500** [emilydolson/python-phylogeny-tracker#4]
**Describe the bug**
`calc_diversity` segfault 

**To Reproduce**
```
Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from phylotrackpy import systematics
>>> sys = systematics.Systematics(lambda x: x)
>>> sys.load_from_file("consolidated.csv", "id", True)
>>> sys
<phylotrackpy.systematics.Systematics object at 0x7f122983d770>
>>> sys.calc_diversity()
zsh: segmentation fault (core dumped)  python3
```
[consolidated.csv](https://github.com/emilydolson/phylotrackpy/files/10929175/consolidated.csv)

**Expected behavior**
no segfault :sweat_smile: 

**Screenshots**

**Computational environment (please complete the following information):**
 - OS: popOS
 - Python version 3.10

**Additional context**
phylotrackpy version 0.1.12

**Comment by emilydolson** [comment]
Is the consolidated.csv file the same one in #5? Assuming yes, I weirdly seem to be unable to reproduce this error (also on popOS and using 3.10). I am getting nan, which is also not what it should be doing, but I don't know why I'm not getting a segfault.

**Comment by emilydolson** [comment]
Oh, figured it out - these are both related to the fact that load_from_file doesn't fill in num_orgs. When @rodsan0 add the functionality for handling num_orgs, both this and #5 should be fixed.

**Comment by rodsan0** [comment]
I think 5dd3e8c over on the mabe-systematics branch should have fixed it? That commit implements the `num_orgs` and `tot_orgs` handling.

**Comment by emilydolson** [comment]
That will fix it for files that contain `num_orgs` and `tot_orgs`, but not for files like this one that don't have either. I'm realizing now that I was wrong about the feature you're currently working on (it gives the user choice over whether to fix depth calculations, right? Somehow I was thinking it was about giving them choice about filling in org counts), so that won't fix this either.

Fundamentally, `calc_diversity` doesn't make sense to do unless you know actual org counts, because its Shannon entropy of the extant taxa. Nothing we could automatically fill in is going to make it make sense. I'm thinking the answer here is to throw an error.

That said, calc_ave_depth could be meaningfully calculated if we assumed all extant taxa has one organism. So I think I might have load make that assumption, while will fix #5

**Comment by emilydolson** [comment]
Okay, this should be fixed in the latest version (although, again, I don't actually recommend using calc_diversity in cases where num_orgs isn't known)
